### DISCUSSION TITLE
Are there examples of using USWDS with Quarto HTML outputs?

### DISCUSSION THREAD

**Discussion by mikemahoney218-usgs** [quarto-dev/quarto-cli#11453] [Q&A]
### Description

Hi all,

I'm in the early stages of a project which would potentially use Quarto to generate HTML pages which would need to use theming and components from the [USWDS](https://designsystem.digital.gov/) SASS library. I've got very little web development experience, and so while I've seen the documentation on [Quarto themes](https://quarto.org/docs/output-formats/html-themes-more.html) and [HTML themes](https://quarto.org/docs/output-formats/html-themes.html#sass-variables) I haven't gotten super far in actually using the USWDS library.

I'm wondering if you all happen to know of any (public) examples of folks doing this integration, or have any other tips on how I might go about trying to get these components into a Quarto webpage.

Thanks!

**Comment by cscheid** [comment]
(👋 !)

Our standard theming system for `html` documents is heavily based on bootstrap, and so I think you'll be fighting an uphill battle to support USWDS directly. This is possible in principle, but would be a big project. Would it be acceptable to have something that looks like their design, but uses the bootstrap structure?

In that case, I would suggest you give https://posit-dev.github.io/brand-yml/ and https://prerelease.quarto.org/docs/authoring/brand.html a read. We're still working on the docs, but I'm pretty excited about `_brand.yml` as a unified way for people to customize Quarto (and Shiny! and, in 2025, plots and tables too!) documents.

**Comment by mikemahoney218-usgs** [comment]
(👋 👋 !)

Thanks for the rapid response!

Unfortunately, I think our compliance people require us to actually use their components, so that we automatically collect any updates they make over time. I'm excited about `_brand.yml` for personal projects, but seems like we probably need to explore a different path forward for this specific task. Thanks!
